usecases/MinimumWorkingExample/upload_scripts/doit_create_types.py

```diff
@@ -22,8 +22,9 @@ def create_required_sample_types(mixture_directory_path: Union[Path, str],
                                  logging_path: Union[Path, str],):
 
     if config['runson'] == 'nodb':
-        _create_logfiles(mixture_sample_type='RAN ON ACTIONS',
-                         emodul_sample_type='RAN ON ACTIONS',
+        _create_logfiles(mixture_sample_type="RAN WITHOUT OPENBIS CONNECTION",
```

Comment:
The "ran on actions" tag was there previously as we were unable to run the workflow on actions. Now we can do it since we have a docker image that runs the database in actions and as such the tag "ran on actions" is not accurate anymore so i changed it to something more appropriate.